Langs/040B.lang

```diff
@@ -0,0 +1,59 @@
+[MetaData]
+Language=Finnish (FI)
+Translator=Nabsi
+Version=2.3.0.1
+[Strings]
+Donate=Lahjoita
+Check for Updates=Etsi Päivityksiä
+Your Windows 11 Compatibility Results are Below=Sinun Yhteensopivuus Tulokset Windows 11 Kanssa Ovat Alapuoella
+Now Reach WhyNotWin11 via https://www.whynotwin11.org/=Nyt voit löytää WhyNotWin11:n osoitteesta https://www.whynotwin11.org/
+Results Based on Currently Known Requirements!=Tulokset Ovat Pohjaisin Tällä Hetkellä Saaduista Vaatimuksista
+Translation by=Kääntänyt
+Architecture (CPU + OS)=Arkkitehtuuri (Processori + Käyttöjärjestelmä)
+Boot Method=Käynnistys Tapa
+CPU Compatibility=Processorin Yhteensopivuus
+CPU Core Count=Processorin Ytimien Määrä
+CPU Frequency=Processori Kellotaajuus
+DirectX + WDDM2=DirectX + WDDM2
+Disk Partition Type=Levyn Osio Tyyppi
+RAM Installed=Ramia Asennettuna
+Secure Boot=Turvallinen Käynnistys
+Storage Available=Tilaa Vapaana
+TPM Version=TMP versio
+Checking...=Varmistetaan...
+64 Bit CPU=x64 Bittinen Processori
+64 Bit OS=x64 Bittinen Käyttöjärjestelmä
+32 Bit CPU=x32 Bittinen Processori
+32 Bit OS=x32 Bittinen Käyttöjärjestelmä
+Unable to Determine=Ei Voida Määrittää
+Unable to Check List=Ei Voida Katsoa Listaa
+Error Accessing List=Ongelma Avatessa Listaa
+Not Currently Listed as Compatible=Ei Tällä Hetkellä Tuoettu
+Listed as Compatible=Listattu Tuetuksi
+Cores=Ytimiä
+Threads=Lanjoka

```

Comment:
This was untranslatable on deepl and on google, both suggested "kierteet" but I think it means threads like on nuts and bolts.

Reply:
well raw translation of Thread is "Lanka" then Threads would be Lankoja that was a mistype indeed.